### ISSUE TITLE
TypeError: Cannot read property 'endsWith' of undefined

### ISSUE THREAD

**Issue by marktani** [hoodsy/graphcool-auth0-example#1]
If I do `yarn start` I immediately get this error output:

```
TypeError: Cannot read property 'endsWith' of undefined
    at new Graphcool (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphcool-binding/dist/Graphcool.js:62:21)
    at GraphQLServer.context (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/server/index.js:17:9)
    at GraphQLServer.<anonymous> (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphql-yoga/dist/src/index.js:135:51)
    at step (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphql-yoga/dist/src/index.js:40:23)
    at Object.next (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphql-yoga/dist/src/index.js:21:53)
    at /Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphql-yoga/dist/src/index.js:15:71
    at Promise (<anonymous>)
    at __awaiter (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphql-yoga/dist/src/index.js:11:12)
    at /Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/graphql-yoga/dist/src/index.js:128:95
    at /Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/apollo-server-core/dist/graphqlOptions.js:52:40
    at step (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/apollo-server-core/dist/graphqlOptions.js:32:23)
    at Object.next (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/apollo-server-core/dist/graphqlOptions.js:13:53)
    at /Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/apollo-server-core/dist/graphqlOptions.js:7:71
    at Promise (<anonymous>)
    at __awaiter (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/apollo-server-core/dist/graphqlOptions.js:3:12)
    at resolveGraphqlOptions (/Users/marktani/projects/playground/beta-2/graphcool-auth0-example/node_modules/apollo-server-core/dist/graphqlOptions.js:43:12)
```

Also see https://github.com/graphcool/graphql-yoga/issues/49.

**Comment by marktani** [comment]
The same problem occurs in the `node-advanced` boilerplate. I expect this to work with `0.5.6`, so it should have been introduced in `0.6.0` or `0.7.0`, but haven't actually verified this.

**Comment by marktani** [comment]
This was fixed in #2.
